R/MIRA.R

```diff
@@ -16,20 +16,21 @@
 #' @importFrom ggplot2 ggplot aes facet_wrap geom_boxplot geom_jitter geom_line
 #' @import BiocGenerics S4Vectors IRanges
 #' @importFrom data.table ":=" setDT data.table setkey fread setnames 
-#'             setcolorder melt setkeyv rbindlist setattr
+#'             setcolorder melt setkeyv rbindlist setattr setorder
 #' @importFrom Biobase sampleNames
 #' @importFrom bsseq getBSseq hasBeenSmoothed
 NULL
 
 
-# Because of some issues with CRAN submission,
+# Because of some issues with CRAN submission, 
```

Comment:
not really clear what this code is doing / why it's necessary ... i didn't see any related NOTES from check ... if you do need it, maybe it's better to put in a utility function (not exported) so that's it at least documented